src/file-map-url-loader.ts

```diff
@@ -23,21 +23,31 @@ import {parseUrl} from 'polymer-analyzer/lib/utils';
 export class FileMapUrlLoader implements UrlLoader {
   root: string;
   files: Map<string, File>;
+  fallbackLoader?: UrlLoader;
 
-  constructor(root: string, files: Map<string, File>) {
+  constructor(
+      root: string,
+      files: Map<string, File>,
+      fallbackLoader?: UrlLoader) {
     this.root = root;
     this.files = files;
+    this.fallbackLoader = fallbackLoader;
   }
 
   // We can always return true because we're just reading paths off a map.
   canLoad(_url: string): boolean {
     return true;
```

Comment:
Okay I updated to use the map and the fallbackLoader (if present) in determining canLoad.